[SaaS] Update getting-started.mdx (#21143)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-for-platforms/cloudflare-for-saas/start/getting-started.mdx

[^1]: <Render file="regional-services" />

:::warning

> If you need to use an A record to point to the SaaS target, you'll need to get [Apex Proxying](/cloudflare-for-platforms/cloudflare-for-saas/start/advanced-settings/apex-proxying/) enabled.

Reviewer: ```suggestion
If you need to use an A record to point to the SaaS target, you will need to get [Apex Proxying](/cloudflare-for-platforms/cloudflare-for-saas/start/advanced-settings/apex-proxying/) enabled.
```

> :::warning
> If you need to use an A record to point to the SaaS target, you'll need to get [Apex Proxying](/cloudflare-for-platforms/cloudflare-for-saas/start/advanced-settings/apex-proxying/) enabled.
> By default, using an A record instead of CNAME to point to the target is **not a supported setup**!
> :::
> 
> 

Reviewer: ```suggestion
:::caution
If your customer needs to use an A record to point to the SaaS target, you will need to get [apex proxying](/cloudflare-for-platforms/cloudflare-for-saas/start/advanced-settings/apex-proxying/). By default, using an A record to point to the target is not a supported setup.
:::

```
